core/src/components/input/input.tsx

```diff
@@ -310,6 +314,19 @@ export class Input implements ComponentInterface {
     return Promise.resolve(this.nativeInput!);
   }
 
+  /**
+   * Emits an `ionChange` event.
+   *
+   * This API should be called for user committed changes.
+   * This API should not be used for external value changes.
+   */
+  private emitValueChange() {
+    const { value } = this;
+    const newValue = value == null ? value : value.toString();
+    this.ionChange.emit({ value: newValue });
```

Comment:
I know this would be a breaking change, but would it be simpler to pass along the native `Event` payload here instead of adding this logic? Would align the API closer to `ionInput` too. Not sure why this behavior was added in the first place though, so worth checking that out too.

Reply:
I like the idea of simplifying this and to align against the `change` event 1:1. I'll do a little discovery as to why this exists. To my knowledge, this payload hasn't been modified in many years, so the likelihood of us needing additional custom properties that requires an object seems low.